### ISSUE TITLE
triangular fibrocartilage complex (TFCC) tear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10721]
**Preferred term label:**
History of triangular fibrocartilage complex (TFCC) tear

**Synonyms**

**Definition (free text, please give PubMed ID)**
Recent or remote history of a tear in the TFCC ligament

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Such a history is important for the diagnosis of Distal radioulnar joint (DRUJ) arthritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Would it be OK to instead add a term for triangular fibrocartilage complex, which could then be used to record the history?

**Comment by MickeySegal** [comment]
OK

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001251
- label: History of triangular fibrocartilage complex tear
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:30725740
- def: Recent or remote history of a tear in the triangular fibrocartilage complex (TFCC). The TFCC is a load-bearing structure between the lunate, triquetrum, and ulnar head. The function of the TFCC is to act as a stabilizer for the ulnar aspect of the wrist. The TFCC is at risk for either acute or chronic degenerative injury.
- comment: Such a history is important for the diagnosis of Distal radioulnar joint (DRUJ) arthritis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10721
- synonyms: History of TFCC tear(NONE)
